server/main.lua

```diff
@@ -245,6 +245,37 @@ QBCore.Functions.CreateCallback('mdt:server:SearchProfile', function(source, cb,
 	return cb({})
 end)
 
+QBCore.Functions.CreateCallback('mdt:server:SearchOfficerProfile', function(source, cb, sentData)
+	if not sentData then  return cb({}) end

```

Comment:
For debugging purposes, something like:

```suggestion
	if not sentData then  
	    print('No data reviewed for searching after officer')
	    return cb({}) 
	end
```

Up to you if you think it makes sense.

Reply:
done <3